src/index.html

```diff
@@ -11,7 +11,187 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group">
+      
+        <legend>Personal information</legend>
```

Comment:
No needs to add empty line between parent and child
```suggestion
      <fieldset class="group">
        <legend>Personal information</legend>
```